_includes/tools/ava/install.md

```diff
@@ -0,0 +1,3 @@
+```console
+$ npm install --save-dev babel-core
```

Comment:
@sindresorhus @hzoo sorry guys didn't catch it :disappointed: 

Should I change it with:
> There is no need to `npm install babel-core`, AVA bundles it already.

Thanks.